src/wymeditor/editor/undo-redo.js

```diff
@@ -22,10 +22,6 @@ WYMeditor.UndoRedo = function (wym) {
 
     undoRedo.wym = wym;
 
-    // https://github.com/mightyiam/object-history
-    undoRedo.history = new WYMeditor.EXTERNAL_MODULES
-        .ObjectHistory(wym.getCurrentState());
```

Comment:
This always produces a first undo point that is lacking selection.

That means that when undoing all the way back to the first point makes the selection go away, meaning the user must place the caret in order to type characters again.